packages/mdc-menu-surface/README.md

```diff
@@ -0,0 +1,225 @@
+<!--docs:
+title: "Menu Surface"
+layout: detail
+section: components
+excerpt: "Material Design menu surface."
+iconId: menu
+path: /catalog/menu-surface/
+-->
+
+# Menu Surface
+
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components.github.io/material-components-web-catalog/#/component/menu">
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/menus.png" width="178" alt="Menus screenshot">
+  </a>
+</div>-->
+
+The MDC Menu Surface component is a reusable surface that appears above the content of the
+page and can be positioned adjacent to an element. Menu Surfaces require JavaScript to properly position
+themselves when opening.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/menus.html">Material Design guidelines: Menus</a>
+  </li>
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components.github.io/material-components-web-catalog/#/component/menu">Demo</a>
+  </li>
+</ul>
+
+## Installation
+
+```
+npm install @material/menu-surface
+```
+
+## Usage
+
+### HTML Structure
+
+```html
+<div class="mdc-menu-surface">
+...
+</div>
+```
+
+### Styles
+
+```css
+@import "@material/menu-surface/mdc-menu-surface";
+```
+
+### JavaScript Instantiation
+
+```js
+import {MDCMenuSurface} from '@material/menu-surface';
+
+const menuSurface = new MDCMenuSurface(document.querySelector('.mdc-menu-surface'));
+```
+
+> See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
+
+
+## Variants
+
+### Anchors and Positioning
+
+#### Anchored To Parent
+
+The menu surface can be positioned to automatically anchor to a parent element when opened.
+
+```html
+<div id="toolbar" class="toolbar mdc-menu-surface--anchor">
+  <div class="mdc-menu-surface">
+  ...
+  </div>
+</div>
+```
+
+#### Anchor To Element Within Wrapper
+
+The menu surface can be positioned to automatically anchor to another element, by wrapping the other element with the anchor class.
+
+```html
+<div class="mdc-menu-surface--anchor">
+  <button id="menu-surface-button">Open Menu Surface</button>
+  <div class="mdc-menu-surface">
+  ...
+  </div>
+</div>
+```
+
+#### Fixed Position
+
+The menu surface can use fixed positioning when being displayed.
+
+```html
+<div class="mdc-menu-surface mdc-menu-surface--fixed">
+...
+</div>
+```
+
+```js
```

Comment:
There should probably be a "Or in JS:" before this, right? You only need one or the other IIUC?